core/controllers/base.py

```diff
@@ -388,7 +388,7 @@ def validate_and_normalize_args(self):
                     handler_class_name))
 
         schema_for_url_path_args = self.URL_PATH_ARGS_SCHEMAS
-        normalized_arg_values, errors = (
+        self.request.route_kwargs, errors = (
```

Comment:
I think from frontend these examples should be in the form of ```encodeUri```.
Also instead of taking an arbitary example for a handler can you pick a real handler which works perfectly by suggested changes.
For reference you can take a look into -- ```PracticeSessionsPageDataHandler```

Reply:
Thanks for the clarification.

A real handler that could utilize this change is `QuestionsListHandler`. It has the following url template: `/questions_list_handler/<comma_separated_skill_ids>?offset=<offset>`, where `comma_separated_skill_ids` is a url path arg we want the normalzied value of. If `QuestionsListHandler` has a schema for it like this:
```
URL_PATH_ARGS_SCHEMAS = {
    'comma_separated_skill_ids ': {
        'schema': {
            'type': 'custom',
            'obj_type': 'JsonEncodedInString'
        }
    }
}
```
We could make a get request from the frontend with a url like this:
 `'/questions_list_handler/' + encodeURIComponent('["id1","id2"]') + '?offset=0'`
And because of the suggested change in this PR we would receive the normalized value of `comma_separated_skill_ids` (a python list) in the backend handler.

Reply:
@kalkas98 Sorry for the delay -- Yes I got your point, can you please write backend test for it.